apps/notifications/src/panel/utils/link-interceptor.js

```diff
@@ -11,43 +9,39 @@ const openComment = ( { siteId, postId, href, commentId } ) => ( {
 	commentId,
 } );
 
-export const interceptLinks = ( event ) => {
+export const interceptLinks = ( event ) => ( dispatch ) => {
 	const { target } = event;
 
 	if ( 'A' !== target.tagName && 'A' !== target.parentNode.tagName ) {
-		return true;
+		return;
```

Comment:
These boolean return values matter only in jQuery click handlers. Here we have a classic DOM/React handler, and we achieve the effect of returning `false` with `event.stopPropagation` + `preventDefault`.